QSB/Animation/AnimationSync.cs

```diff
@@ -101,8 +101,17 @@ public void InitRemote(Transform body)
         private void OnBecomeGrounded() => SendTrigger(AnimTrigger.Grounded);
         private void OnBecomeUngrounded() => SendTrigger(AnimTrigger.Ungrounded);
 
-        private void OnSuitUp() => SendTrigger(AnimTrigger.SuitUp);
-        private void OnSuitDown() => SendTrigger(AnimTrigger.SuitDown);
+        private void OnSuitUp()
+        {
+            PlayerRegistry.LocalPlayer.UpdateState(State.Suit, true);
+            SendTrigger(AnimTrigger.SuitUp);
+        }
+
+        private void OnSuitDown()
+        {
+            PlayerRegistry.LocalPlayer.UpdateState(State.Suit, false);
```

Comment:
This is already handled in the OnReceiveLocal of the suit event, not sure where is the best place to keep it though

Reply:
Removed from OnReceiveLocal, should set state right away instead of when we get the message back